routes/rss.js

```diff
@@ -23,12 +25,36 @@ async function combinedRSSFeedRoute(req, res) {
 
           const author = blogRegistry[url]['chosenName'];
           const blogUrl = url;
-          const publishedAt = new Date(feed.items[0].pubDate);
 
-          return feed.items.map((item) => ({
-            item,
-            metadata: { author, blogUrl, publishedAt },
-          }));
+          return feed.items.map((item) => {
+            let blogPostUrl = item.link || '';
+            const publishedAt = item.pubDate
+              ? new Date(item.pubDate)
+              : new Date();
+
+            // Sometimes, it seems that `item.link` is not a full URL, so we
+            // need to join it with the blog URL.
+            if (!blogPostUrl.startsWith('http')) {
+              blogPostUrl = path.join(blogUrl, blogPostUrl);
```

Comment:
Yeah, this is part of the reason I ended up reformatting the `blogRegistry` to be an object of objects instead of simple JSON. I ended up pulling the url straight from there (a la https://github.com/ibrand/yearoftheblog.us/blob/main/index.js#L46)

This looks good!